### PR TITLE
docs : Enhanced documentation by adding a diagram

### PR DESCRIPTION
**Pull request by AbdulMalikDev**

### CODE REVIEW

**Comment by harchani-ritik** [comment]
@AbdulMalikDev 
You can state the communication protocol between rutorrent and torrent as XML-RPC in the diagram.

**Comment by AbdulMalikDev** [comment]
@harchani-ritik Thank you for the reply. I'll add the text "XML-RPC" on the line between rutorrent and rtorrent. Let me know if that would be sufficient.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ ruTorrent ⇒ A web interface for rtorrent that uses that API. It also does othe
 
 Thus, our Flutter application talks with ruTorrent's backend service to provide a native interface.
 
+![Diagram](https://github.com/AbdulMalikDev/rutorrent-flutter/blob/master/assets/docs/ruTorrent%20Flutter%20Application%20Diagram.png)
+
```
Comment (tusharojha) [inline README.md:32]: Using an absolute path as URLs is not a good approach if the file lies in the same repository.
I would suggest the URL as `./assets/docs/<image name>.png`.

**Review by tusharojha — CHANGES_REQUESTED** [review]
Suggesting a small change.

File: README.md (hunk 1 above)
Comment (AbdulMalikDev) [reply README.md:32]: Thank you for sharing this approach, will definitely keep this in mind and reflect this change in upcoming PRs.

**Pull request merged**